packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -112,6 +114,8 @@ export function autocomplete<TItem extends BaseItem>(
     refresh: autocomplete.value.refresh,
   };
 
+  const html = htm.bind<VNode>(props.value.renderer.renderer.createElement);
```

Comment:
This is an edge case, but the `html` function can get outdated if the `renderer` option changes. If we want to support this, we need to use `reactive()`.

Reply:
As in, pointing one of the renderer properties to a different functions after instantiating Autocomplete? Because I don't think it even works today.

Reply:
Doing so won't rebind the `html` function passed to the templates:

```jsx
import { createElement, Fragment, render } from 'preact';

const search = autocomplete({ /* ... */ })

search.update({
  renderer: { createElement, Fragment, render }
});
```

Our props are reactive and we get the `renderer` from the props, they it should work. Since in this PR `html` is not reactive (it's in the function scope), the value won't be synced with `renderer` if it changes.

Reply:
Oh right, I forgot about the `update` API 🤦‍♀️

Addressed in 0554e37f.

Reply:
Tracked here https://github.com/algolia/autocomplete/issues/934